Base62/Base62Converter.cs

```diff
@@ -37,10 +37,11 @@ public string Decode(string value)
                 arr[i] = (byte)characterSet.IndexOf(value[i]);
             }
 
-            return Decode(arr);
+            var converted = Decode(arr);
+            return Encoding.UTF8.GetString(converted, 0, converted.Length);
         }
 
-        private string Encode(byte[] value)
+        public string Encode(byte[] value)
```

Comment:
Last thing, just noticed here we should also expose this with the signature:
```suggestion
        public byte[] Encode(byte[] value)
```

Then let the string method handle the conversion from byte array to string.  Otherwise, all looks good. After you make the changes I'll get it in with a few other changes and prepare a release

Reply:
Yes, this makes total sense. I commited these changes.